raiden/tests/unit/test_transfer.py

```diff
@@ -158,16 +157,17 @@ def test_transfer(raiden_network):
     assert isinstance(a1_recv_messages[0], DirectTransfer)
 
 
+@pytest.mark.timeout(10)
```

Comment:
it's not using geth, 10 seconds should be more than enough, tested on a slow machine.